features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/AsyncApiUI.jsx

```diff
@@ -22,16 +22,52 @@ import Api from '../../../../data/api';
 import Progress from '../../../Shared/Progress';
 import WebhookSubscriptionUI from './WebhookSubscriptionUI';
 import GenericSubscriptionUI from './GenericSubscriptionUI';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import { makeStyles } from "@material-ui/core/styles/index";
+import CONSTANTS from 'AppData/Constants';
+import Alert from 'AppComponents/Shared/Alert';
+import { injectIntl } from 'react-intl';
```

Comment:
since this is a functional component, Shall we use 
```
useIntl()
```
hook instead
https://formatjs.io/docs/react-intl/api/#useintl-hook